application.py

```diff
@@ -73,6 +73,31 @@ def login():
         return render_template("registration.html", message="you haven't registered please register first")
 
 
+@app.route("/search")
+def search():
+    req = request.form
+    searchword = req.get(searchword)
+    books = search.get_books(searchword)
+    if (len(books) == 0):
+        return render_template("search_result.html", message="No reults found with the given keyword")
+    else:
+        return render_template("search_result.html", result=books)
+
+
+def get_books(searchword):
+    totalbooks=[]
+    search = "%{}%".format(searchword)
+    books_a = Books.query.filter(Books.author.like(search)).all()
+    books_t = Books.query.filter(Books.title.like(search)).all()
+    books_isbn = Books.query.filter(Books.isbn.like(search)).all()
+    books_year = Books.query.filter(Books.year.like(search)).all()
```

Comment:
can use "OR" keyword and write the query in one line

Reply:
Updated the query ma'am.